templates/slurm/header_tnt.ftl

```diff
@@ -48,7 +48,8 @@ export mydate_start
 #
 <#noparse>
 runName=$(basename $(cd ../ && pwd ))
-MC_failedFile="${logsDirectory}/${runName}.pipeline.failed"
+projectName=$(basename $(cd ../../ && pwd ))
+MC_failedFile="${logsDirectory}/${projectName}/${runName}.pipeline.failed"
```

Comment:
Please apply the same fix also to the `header.ftl` (without tnt) to keep them in sync.